deploy/README.md

```diff
@@ -27,8 +27,10 @@ This page has instructions for collecting Kubernetes logs, metrics, and events;
             - [Trim and relabel metrics](#trim-and-relabel-metrics)
         - [Custom Metrics](#custom-metrics)
             - [Step 1: Expose a `/metrics` endpoint on your service](#step-1-expose-a-metrics-endpoint-on-your-service)
-            - [Step 2: Setup a service monitor so that Prometheus pulls the data](#step-2-setup-a-service-monitor-so-that-prometheus-pulls-the-data)
-            - [Step 3: Update the prometheus-overrides.yaml file to forward the metrics to Sumo.](#step-3-update-the-prometheus-overridesyaml-file-to-forward-the-metrics-to-sumo)
+            - [Step 2: Set up a service monitor so that Prometheus pulls the data](#step-2-set-up-a-service-monitor-so-that-prometheus-pulls-the-data)
+            - [Step 3: Step 3: Create a new HTTP source in Sumo Logic.](#step-3-create-a-new-http-source-in-sumo-logic)
```

Comment:
`Step 3: ` is written twice here

Reply:
Fixed

Reply:
did you commit the change? still written twice

Reply:
hrmm its weird, i don't see this locally